docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[troubleshooting]]
+== Troubleshooting
+
+Below are some resources and tips for troubleshooting and debugging the
+python agent.
+
+[float]
+[[easy-fixes]]
+=== Easy Fixes
+
+Before you try anything else, go through the following sections to ensure that
+the agent is configured correctly. This is not an exhaustive list, but rather
+a list of common problems that users run into.
+
+[float]
+[[debug-mode]]
+==== Debug Mode
+
+Most frameworks support a debug mode. Generally, this mode is intended for
+non-prod environments and provides detailed error messages and logging of
```

Comment:
Generally, I'd try to avoid abbreviations like "prod" in documentation. You never know if the reader knows the abbreviation or maybe interprets it differently than intended.